src/openrct2/localisation/Date.h

```diff
@@ -61,8 +61,10 @@ extern openrct_timeofday gRealTimeOfDay;
 sint32 date_get_month(sint32 months);
 sint32 date_get_year(sint32 months);
 sint32 date_get_total_months(sint32 month, sint32 year);
+sint32 cmdline_for_date(const utf8 **argv, sint32 argc);
```

Comment:
Note to self: forgot to remove this line. Should probably mark `cmdline_for_date` as static in Console.cpp, too.